src/main/java/com/getindata/connectors/http/internal/table/lookup/HttpLookupTableSourceFactory.java

```diff
@@ -164,6 +130,7 @@ private List<String> convertArguments(String arguments) {
         return Arrays.stream(arguments.split(";")).map(String::trim).collect(Collectors.toList());
     }
 
+    // TODO verify this since we are on 1.15 now.
```

Comment:
What do we want to verify?

Reply:
do I still need to back-port code below from Flink 1.15 :)

Reply:
![image](https://user-images.githubusercontent.com/7932805/184414017-56e2701a-3451-41bf-a359-32cffd642569.png)